core/services/ardupilot_manager/setup.py

```diff
@@ -17,9 +51,12 @@
         "appdirs==1.4.4",
         "packaging==20.4",
         "smbus2==0.3.0",
+        "starlette == 0.13.6",
```

Comment:
Can you be consistent with the style here between == ?

Reply:
Fixed